packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -1,45 +1,374 @@
 <template>
-    <component
-        :is="componentVersion"
-        v-bind="data"
-        :flags="flags"
-        :error-boundary="errorBoundary">
-        <template
-            v-for="(_, slotName) in $slots"
-            v-slot:[slotName]>
-            <slot :name="slotName" />
-        </template>
-    </component>
+    <a
+        :href="url"
+        :class="[
+            $style['c-restaurantCard'], {
+                [$style['c-restaurantCard--listItem']]: isListItem,
+                [$style['c-restaurantCard--hasImg']]: !!imgUrl
+            }]"
+        data-test-id="restaurantCard-component"
+        @click="handleClick">
```

Comment:
Although there is a bit of a mixture, convention in fozzie looks to mostly be kebab case for test ids so might want to change this to `data-test-id="restaurant-card-component"`

Reply:
Absolutely - the next PR is to sync the test IDs between searchweb and fozzie so I'll be updating all of them in that change :)

Reply:
Yeh, data attributes are case insensitive, so should always be lowercase (which is why it's the only place I think we don't use camelCase)